### ISSUE TITLE
"Engineer not present in active vessel"

### ISSUE THREAD

**Issue by LolscopeDev2022** [UmbraSpaceIndustries/MKS#1574]
Hi there! I kinda was trying to use KonFabricator to make an Agricultural module for example. But instead, it shows me "**Engineer not present in active vessel**" message. I have 2 engineers and 1 pilot on my vessel. Even when I moved one of the engineers into KonFabricator, it still showed me this message. And because of that, I can't Konstruct any new parts, either enable Konstruction mode for EVA engineers as well.
![mks konstruction bug](https://user-images.githubusercontent.com/67801031/225562263-dc22ffbc-d82b-4cf1-aa68-f6e8b45ee8b1.png)
![mks konstruction bug2](https://user-images.githubusercontent.com/67801031/225562271-494ea3b0-fc8d-42d7-8d4a-27c8388eb163.png)

**Comment by helitheone** [comment]
Yes, I have the problem, unfortunately, is due to the language set. It works fine when set to english. Unfortunately, that's not an option for me

**Comment by OctaAcid** [comment]
If you still interested in this mod
https://github.com/UmbraSpaceIndustries/Konstruction/issues/96
This issue on Konstruction mod showed the reason of it and provides a solution of replacing the name of kerbals' job into its English version instead of changing the language for the entire game.
